main.tf

```diff
@@ -0,0 +1,67 @@
+module "s3_bucket" {
```

Comment:
This s3 bucket is optional, and only necessary if the user is specifying a crl_configuration for the CA. We can use count to reflect that condition. something like:

```
count = var.private_certificate_authority.revocation_configuration.crl_configuration != null ? 1 : 0
```